packages/telemetry/shim.js

```diff
@@ -0,0 +1,66 @@
+/* global globalThis */
+import { MeterProvider } from '@opentelemetry/metrics';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+
+const configNames = [
+  'OTEL_EXPORTER_PROMETHEUS_HOST',
+  'OTEL_EXPORTER_PROMETHEUS_PORT',
+];
+
+const getConfig = env => {
+  const config = new Map();
+  configNames.forEach(name => {
+    const value = env[name];
+    if (value !== undefined) {
+      config.set(name, value);
+    }
+  });
+  return config;
+};
+
+// This is named getMetricProvider instead of getMeterProvider to avoid horrible
+// confusion with SwingSet metering.
+globalThis.getMetricProvider = (console, env) => {
```

Comment:
I’ve got a strong sense that the thing we want on `globalThis` should just be a `metrics`, `meter`, or `telemetry` namespace that closes over whatever `console` or `process.env` was present at time of initialization and that `getMetricsProvider` should be hidden by a thin layer of shim.

Reply:
Eliminated use of `globalThis`.  It was misguided.